application/src/main/java/org/togetherjava/tjbot/commands/example/CommandExample.java

```diff
@@ -13,12 +12,15 @@
 import net.dv8tion.jda.api.interactions.components.Component;
 import net.dv8tion.jda.api.requests.restaction.CommandCreateAction;
 import org.jetbrains.annotations.NotNull;
+import org.togetherjava.tjbot.commands.AbstractCommand;
+import org.togetherjava.tjbot.commands.Command;
```

Comment:
Why do you need to import this here? It forces you to qualify some names further down but I don't see where you use it?

Reply:
Didn't notice that, got probably handled by IntelliJ resulting in a fully qualified name within the code.